pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -0,0 +1,629 @@
+// api.pb.go could be generate by hack/update-generate-runtime.sh
+syntax = 'proto2';
+
+package runtime;
+
+// Runtime service defines the public APIs for remote container runtimes
+service RuntimeService {
+    // Version returns the runtime name, runtime version and runtime API version
+    rpc Version(VersionRequest) returns (VersionResponse) {}
+    // Status returns error if the runtime is unhealthy; nil otherwise
+    rpc Status(StatusRequest) returns (StatusResponse) {}
```

Comment:
Hm...I'm not sure if we need to have `Status`. kubelet can call `Version` to check if runtime is responsive. Internally, `Status` just maps to some runtime call.